CIP-Liesenfelt-Shelleys_Voltaire_decentralization_update/README.md

```diff
@@ -164,41 +196,39 @@ Large differences between k-parameter and the k-effective of the network represe
 
 ## Statement of Principles
 
-“Principles matter.” -CH
+> “Principles matter.” -CH
 
 1. Everybody in the community should be treated fairly from tiny starfish delegators (1-2k₳) to massive blue whales (>100M₳) and exchanges.
 2. Everybody in the community should have the opportunity to on average earn the same yield and there should not be two classes of stakeholders.
-3. There should be a very clear cause and effect relationship between the (a0,k) input parameters and the resulting K-effective decentralization.
-4. The a0 input parameter must require, not incentivize, pool operators to pledge to support community delegation and sybil protection.
+3. There should be a very clear cause and effect relationship between the $(a0,k)$ input parameters and the resulting K-effective decentralization.
+4. The $a0$ input parameter must require, not incentivize, pool operators to pledge to support community delegation and sybil protection.
 5. The decentralization result will be quantified in terms of group decentralization of block production, not stake saturation relative to pools.
 6. Implementation should be smooth, easy, clear, and beneficial for all stakeholders and operators.
 7. A new reward equation should be computationally simple and elegant.
 
 ## The Proposed Reward Formula
 
-The proposed reward retains the function of k for limiting rewards based on stake but repurposes the a0 parameter for enforcing reward limits based on pledge leverage. The equation equally balances both reward parameters. Instead of a0 ranging from 0.0 to infinity the a0 parameter is intended to range from 10,000.0 down to 1.0. An a0 value of 100.0 would require pools to pledge 1.0% of stake and an a0 of 1.0 would require all pools to be 100.0% pledged.
-
-<img src="new equation.png" width="400">
+The proposed reward retains the function of $k$ for limiting rewards based on stake but repurposes the $a0$ parameter for enforcing reward limits based on pledge leverage. The equation equally balances both reward parameters. Instead of $a0$ ranging from 0.0 to infinity the $a0$ parameter is intended to range from 10,000.0 down to 1.0. An $a0$ value of 100.0 would require pools to pledge 1.0% of stake and an $a0$ of 1.0 would require all pools to be 100.0% pledged.
 
-R = ( reserve * rho + fees )( 1.0 - tau )
+$$ R = ( reserve * \rho + fees )( 1 - \tau ) $$
 
-r( sigma, lambda ) = R * min{ sigma, a0 * lambda, 1/k }
+$$ r( \sigma, \lambda ) = R * min\bigg\{ \sigma, a0 * \lambda, \frac{1}{k} \bigg\} $$
 
```

Comment:
There seems to be a render failure here.

Reply:
going to revert back to your images, no easy fix, Github doesn't fully support LaTeX yet.  I was tricked because it renders fine in VS Code and Jupyter Notebooks...

Reply:
going to have to revert back to your images, github does not support too fancy of math yet (LaTeX).

Reply:
found a way to fix it, was not easy. long story short, github does not support latex fully.  Needs some custom tweaking.